### PR TITLE
Return player id for players:add and players:remove

### PR DESCRIPTION
**Pull request by taitfuller**
Oversight omitted from #68

### CODE REVIEW

File: backend/src/handlers/player.handler.ts [hunk 1]
```diff
@@ -86,7 +86,7 @@ export const playerJoin = async (io: Server, socket: Socket): Promise<void> => {
     const player = await getPlayer(game.gameCode, playerId, game);
     if (player.new) {
       await initialisePlayer(game, playerId);
-      socket.to(game.gameCode).emit("players:add", player.nickname);
+      socket.to(game.gameCode).emit("players:add", player.id, player.nickname);
     }
     socket.data.nickname = player.nickname;
```
Comment (RawiriHohepa) [inline backend/src/handlers/player.handler.ts:91]: What does this do? Do you need to add one for id?

Comment (taitfuller) [reply backend/src/handlers/player.handler.ts:91]: This returns the args `player.id` and `player.nickname` for the event `players:add`. This corresponds to the frontend implementing the following:

`socket.on("players:add", (id, nickname) => { });`

Comment (RawiriHohepa) [reply backend/src/handlers/player.handler.ts:91]: I was referring to ``socket.data.nickname = player.nickname;``

Comment (taitfuller) [reply backend/src/handlers/player.handler.ts:91]: Ah gotcha - if you take a look at `auth.handler.ts` `socket.data.gameCode` & `socket.data.playerId` are already set 👌🏼

**Review by RawiriHohepa — APPROVED** [review]
Looks good, thanks for fixing this

**Pull request merged**